packages/expo-updates/src/__tests__/Updates-test.native.ts

```diff
@@ -0,0 +1,47 @@
+import * as Updates from '../Updates';
+import ExpoUpdates from '../ExpoUpdates';
+
+const fakeManifest = {
+  id: '@jester/test-app',
+  sdkVersion: '36.0.0',
+};
+
+it('returns the proper object in checkForUpdateAsync if an update is available and manifest is returned', async () => {
+  const mock = jest.fn();
+  ExpoUpdates.checkForUpdateAsync = mock;
```

Comment:
Shouldn't this already be set to a mock function by the Jest preset? If the intent here is to clear the mock's state, call `ExpoUpdates.checkForUpdateAsync.mockReset()` in afterEach.